<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>878</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>Sitka Small</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Клиенты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Sitka Small</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Счета</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Sitka Small</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Транзакции</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Sitka Small</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Кредиты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="font">
        <font>
         <family>Sitka Small</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Карты</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="font">
        <font>
         <family>Sitka Small</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Сотрудники</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Sitka Small</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="text">
        <string>Журнал активности</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QStackedWidget" name="stackedwitged">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>361</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_8"/>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>291</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="clients">
       <item>
        <widget class="QPushButton" name="pushButton_8">
         <property name="font">
          <font>
           <family>Sitka Small</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Добавить клиента</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_9">
         <property name="font">
          <font>
           <family>Sitka Small</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Редактировать клиента</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="font">
          <font>
           <family>Sitka Small</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Удалить клиента</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>291</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="accounts">
       <item>
        <widget class="QPushButton" name="pushButton_12">
         <property name="font">
          <font>
           <family>Sitka Small</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Добавить счет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_11">
         <property name="font">
          <font>
           <family>Sitka Small</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Редактировать счет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_13">
         <property name="font">
          <font>
           <family>Sitka Small</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Закрыть счет</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="verticalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>291</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="transactions">
       <item>
        <widget class="QPushButton" name="pushButton_14">
         <property name="text">
          <string>Создать транкзакцию</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_16">
         <property name="text">
          <string>Поиск транзакций</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="verticalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>291</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="credits">
       <item>
        <widget class="QPushButton" name="pushButton_15">
         <property name="text">
          <string>Выдать кредит</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_17">
         <property name="text">
          <string>Редактировать кредит</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_18">
         <property name="text">
          <string>Закрыть кредит</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QWidget" name="verticalLayoutWidget_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>291</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="cards">
       <item>
        <widget class="QPushButton" name="pushButton_19">
         <property name="text">
          <string>Добавить карту</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_20">
         <property name="text">
          <string>Привязать карту к счету</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_21">
         <property name="text">
          <string>Удалить карту</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QWidget" name="verticalLayoutWidget_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="employees">
       <item>
        <widget class="QPushButton" name="pushButton_22">
         <property name="text">
          <string>Добавить сотрудника</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_23">
         <property name="text">
          <string>Редактировать данные о сотруднике</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_24">
         <property name="text">
          <string>Удалить сотрудника</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_25">
         <property name="text">
          <string>Управление ролями</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QWidget" name="verticalLayoutWidget_8">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="activitylog">
       <item>
        <widget class="QPushButton" name="pushButton_29">
         <property name="text">
          <string>Поиск активности</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabwidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>380</y>
      <width>751</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="font">
      <font>
       <family>Sitka Small</family>
       <pointsize>6</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Клиенты</string>
     </attribute>
     <widget class="QTableView" name="clientstableview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="clientscombobox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_27">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Счета</string>
     </attribute>
     <widget class="QTableView" name="accountstableview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="accountscombobox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_28">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Транзакции</string>
     </attribute>
     <widget class="QTableView" name="transactionstableview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="transactionscombobox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="transactionsdateEditStart">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="transactionsdateEditEnd">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_30">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Кредиты</string>
     </attribute>
     <widget class="QTableView" name="creditstableview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="creditscombobox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_31">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Карты</string>
     </attribute>
     <widget class="QTableView" name="cardstableview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cardscombobox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_32">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Сотрудники</string>
     </attribute>
     <widget class="QTableView" name="employeestableview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="empoloyeescombobox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_33">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Журнал активности</string>
     </attribute>
     <widget class="QTableView" name="activitylogtableview">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>461</width>
        <height>141</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="activitylogcombobox">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="activitylogsdateEditStart">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>50</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="activitylogsdateEditEnd">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>90</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_34">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_26">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Личный кабинет</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>878</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
